.circleci/bash_env.sh

```diff
@@ -6,5 +6,5 @@ export GIT_DIRTY=$(test -n "`git status --porcelain`" && echo "+CHANGES" || true
 export GIT_IMPORT=github.com/hashicorp/consul/version
 # we're using this for build date because it's stable across platform builds
 # the env -i and -noprofile are used to ensure we don't try to recursively call this profile when starting bash
-export GIT_DATE=$(env -i /bin/bash --noprofile -norc ${CIRCLE_WORKING_DIRECTORY}/build-support/scripts/build-date.sh)
```

Comment:
this was not expanding correctly: https://discuss.circleci.com/t/circle-working-directory-doesnt-expand/17007

this caused all CI runs to output:
```
~/project/build-support/scripts/build-date.sh: No such file or directory
```